src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task{
```

Comment:
I think it is better stylistically if you put a space between Task and the curly bracket